source/features/parse-backticks.tsx

```diff
@@ -6,7 +6,7 @@ import {parseBackticks} from '../libs/dom-formatters';
 function init(): void {
 	for (const title of select.all(`
 		[aria-label="Issues"][role="group"] .js-navigation-open,
-		.commit-title .js-navigation-open
+		.message
```

Comment:
This seems rather generic. Needs testing to make sure it doesn’t target unrelated elements

Reply:
I suspect the `message` class is the one we should use. It will e.g. also get the code styling to work in the file list:

![image](https://user-images.githubusercontent.com/22477950/71374100-b8edef80-25b9-11ea-993a-174b31b5f992.png)